.github/workflows/coverage.yml

```diff
@@ -61,7 +61,7 @@ jobs:
 
       - name: Calculate base code coverage
         run: |
-          export CUR_COVER=$(npx tap | xargs -0 node -p 'parseInt(process.argv[1].split("\nAll files")[1].split("|")[1].trim(), 10)')
+          export CUR_COVER=$(npx tap test | xargs -0 node -p 'parseInt(process.argv[1].split("\nAll files")[1].split("|")[1].trim(), 10)')
```

Comment:
what does adding `test` here do?

Reply:
it points to the `./test` folder only, otherwise `tap` would not also pick the `./examples` folder during unit tests